### ISSUE TITLE
download-wasm job can unexpectedly fail in CI

### ISSUE THREAD

**Issue by james-chf** [anoma/anoma#1208]
eg https://ci.heliax.dev/anoma/anoma/4243/2/5

```
+ sh scripts/ci/download-wasm.sh

An error occurred (404) when calling the HeadObject operation: Not Found
```

I'm not sure why this failed, the `build-wasm` job hadn't finished yet so perhaps wasms were not uploaded? On a rerun of the CI, this `download-wasm` job passed (https://ci.heliax.dev/anoma/anoma/4244/2/5)

**Comment by Fraccaman** [comment]
yes, `download-wasm` depends on `build-wasm` and sometimes `build-wasm` takes a little bit more time to compile than `download-wasm`. We can't do it better atm, we will have to rework the CI.
